package.json

```diff
@@ -256,7 +256,7 @@
   },
   "devDependencies": {
     "@types/mocha": "^2.2.42",
-    "@types/node": "^8.10.25",
+    "@types/node": "^8.10.66",
```

Comment:
Is this change unnecessary?

Reply:
I have no clue. I really stumbled my way through this. I'll remove this